src/CMakeLists.txt

```diff
@@ -377,6 +379,21 @@ ENDFOREACH (name)
 # Make a directory for sac2c output
 FILE (MAKE_DIRECTORY "${DLL_BUILD_DIR}/${TARGET_ENV}/${SBI}")
 
+# This functions tries to match a module name with its stored
+# path information in a list. It returns on the first result, or
+# returns empty.
+FUNCTION (MATCH_NAME_IN_LIST list name ret)
+    #MESSAGE (DEBUG "Searching for ${name} in list...")
```

Comment:
These commented debug messages look ugly.  If you want to keep them maybe add a DEBUG macro that would conditionally print the message in case some global debug flag is on.

Reply:
The `DEBUG` type for the message already does this, these are only printed when cmake is run using: `--log-level DEBUG` or higher. I forgot thought that I commented these... I might change the message slightly.